bundles/test/org.eclipse.smarthome.test/src/main/java/org/eclipse/smarthome/test/java/JavaOSGiTest.java

```diff
@@ -64,7 +64,7 @@ public void bindBundleContext() {
      *
      * @return bundle context
      */
-    private BundleContext getBundleContext() {
+    public BundleContext getBundleContext() {
```

Comment:
I did not notice that `bundleContext` is already `protected`. I just saw that the method `getBundleContext` existsted. But I agree that its better to use the `protected` member variable. I have amended my commit to fix it.